client/state/data-layer/wpcom-http/utils.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns response data from an HTTP request success action if available
+ *
+ * @param {Object} action may contain HTTP response data
+ * @returns {any|null} response data if available
```

Comment:
> is the syntax valid or does it need to be surrounded with parentheses

I'm pretty sure you're right about `any` but this seems to work fine without parenthesis. though in this case I should have used `{?*}` to indicate the gullibility of the type. I'm recently learning more about JSDoc types